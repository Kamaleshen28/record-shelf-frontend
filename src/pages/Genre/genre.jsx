import React from 'react'
import Header from '../../components/Header/header'
import AllGenre from '../../components/AllGenre/allGenre'

export default function Genre(){

    return(
        <div>
            <Header />
            <AllGenre />
        </div>
    )
}