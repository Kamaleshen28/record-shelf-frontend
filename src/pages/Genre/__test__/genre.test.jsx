import { render} from '@testing-library/react';
import React from 'react';
import { useNavigate } from 'react-router-dom';
import Genre from '../genre';

jest.mock('react-router-dom');

describe('allSongs', () => {
    it('should render allSongs correctly', () => {
        const {asFragment} = render(<Genre />);
        expect(asFragment()).toMatchSnapshot();
    });
});