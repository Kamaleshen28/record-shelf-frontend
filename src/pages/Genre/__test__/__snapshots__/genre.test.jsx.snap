// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`allSongs should render allSongs correctly 1`] = `
<DocumentFragment>
  <div>
    <div
      class="header"
    >
      <p>
        My 
        <span
          class="record-header"
        >
          Record
        </span>
         Shelf
      </p>
    </div>
    <div
      class="body-dummy"
    >
      <div
        class="body-all-songs"
      >
        <div>
          <div
            class="body-header"
          >
            <p
              class="all-songs-text"
            >
              genres
            </p>
            <img
              alt="Not Found"
              class="grid-icon"
              data-testid="grid-toggle-icon"
              src="icon-grid.svg"
            />
          </div>
          <div
            class="all-genre"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
